core/services/ocr2/validate/config.go

```diff
@@ -45,7 +45,9 @@ func ToLocalConfig(ocr2Config OCR2Config, insConf InsecureConfig, spec job.OCR2O
 		ccTrackerPollInterval = ocr2Config.ContractPollInterval()
 	}
 	lc := types.LocalConfig{
+		DefaultMaxDurationInitialization:   30 * time.Second,
 		BlockchainTimeout:                  blockchainTimeout,
+		ContractConfigLoadTimeout:          blockchainTimeout,
```

Comment:
- [ ] todo confirm

Reply:
cc @secure-lorenz

Reply:
@kaleofduty

Reply:
`DefaultMaxDurationInitialization` looks like a good default, though I could imagine that Data Streams would want something lower. Is it possible for products to override these?

Re `ContractConfigLoadTimeout`, what is the default for `blockchainTimeout`?

Reply:
The `blockchainTimeout` default is 20s, and it can be overridden globally from TOML, or per job via the spec. `DefaultMaxDurationInitialization` cannot be overridden, but that could easily be added.

Reply:
`blockchainTimeout` sgtm then.
Let me summon @samsondav re Data Streams.